.github/workflows/validate_examples.yml

```
@@ -0,0 +1,19 @@
+on:
+  - pull_request
+  - push
+
+jobs:
+  validate-examples:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332  #v4.1.7
```

```suggestion
      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  #v4.2.2
```

Update the Actions version